qa-ibd.planx-pla.net/etlMapping.yaml

```diff
@@ -68,3 +68,21 @@ mappings:
           - name: year
             src: year
             fn: set
+  - name: study_viewer_index
+    doc_type: study_viewer_file
+    type: aggregator
+    root: project
+    props:
+      - name: name
+      - name: code
+      - name: data_available_for_request
```

Comment:
as mentioned on slack, i don't think we need to add this `data_available_for_request` field to the dictionary because there is no use case in IBD for disabling access requests for specific projects